### PR TITLE
Changed next button to anchor

### PR DESCRIPTION
**Pull request by tladendo**
fyi @vietnogi @indytechcook

### CODE REVIEW

**Comment by indytechcook** [comment]
looks good

**Comment by vietnogi** [comment]
i don't think we need to changed to anchor.  if the problem is that it is submitting the form, then it has already been solved from my refactor

**Comment by tladendo** [comment]
@vietnogi -- theo asked that i change the 'cancel' button to an anchor.

**Comment by vietnogi** [comment]
@tladendo 
whoa? how come?

**Comment by tladendo** [comment]
@vietnogi -- for styling reasons. my impression is that it's temporary. i think it's a little silly but it's like a 4-line change.

File: src/client/app/submit-work/about-project/styles/brief.scss [hunk 1]
```diff
@@ -8,8 +8,13 @@
...
     @include main-restricted();
 
+
     .character-count {
       text-align: right;
     }
   }
+  a.cancel {
+    display: block;
+    margin-top: 15px;
```
Comment (vietnogi) [inline src/client/app/submit-work/about-project/styles/brief.scss]: Can you use `margin-top: $margin-18`

Comment (tladendo) [inline src/client/app/submit-work/about-project/styles/brief.scss]: sure i'll change

**Comment by vietnogi** [comment]
@tladendo , hope you don't get offended, but I added https://github.com/appirio-tech/ap-work-client/commit/cd065387a86215a91afb8752ea642810dff6ec32, to this PR.  The cancel should behave as a button instead of a link.  I removed the background color for all cancel buttons.

**Pull request merged**